scipy/optimize/_minpack_py.py

```diff
@@ -733,6 +738,12 @@ def curve_fit(f, xdata, ydata, p0=None, sigma=None, absolute_sigma=False,
     Box constraints can be handled by methods 'trf' and 'dogbox'. Refer to
     the docstring of `least_squares` for more information.
 
+    References
+    ----------
+    [1] K. Vugrin et al. Confidence region estimation techniques for nonlinear
+        regression in groundwater flow: Three case studies. Water Resources
+        Research, Vol. 43, W03423, doi:10.1029/2005WR004804
```

Comment:
Oops since doc builds aren't working I didn't check the render, but this won't look right. I think it's supposed to be
```suggestion
        Research, Vol. 43, W03423, :doi:`10.1029/2005WR004804`
```